Migration guide into README. (#96)
brooklyn-data/dbt_artifacts: README.md

file_format = (type = 'JSON')
```

## Running the models

> The models will be picked up on your next `dbt run` command. You can also run the package specifically with `dbt build -s dbt_artifacts`.

Reviewer: ```suggestion
Run `dbt build` (or `dbt run`). You can also run the package specifically with `dbt build -s dbt_artifacts`.
```
